### PR TITLE
Optional blending

### PR DESCRIPTION
**Pull request by themuflon**

### CODE REVIEW

File: loom/graphics/gfxQuadRenderer.cpp [hunk 1]
```diff
@@ -288,15 +299,17 @@ void QuadRenderer::initializeGraphicsResources()
 
     lmLogInfo(gGFXQuadRendererLogGroup, "Initializing Graphics Resources");
 
+    GL_Context* ctx = Graphics::context();
```
Comment (bengarney) [inline loom/graphics/gfxQuadRenderer.cpp]: Does this gain us anything in perf or just code count?

**Comment by bengarney** [comment]
This looks pretty good, one nit and waiting on green.

File: loom/graphics/gfxQuadRenderer.cpp (hunk 1 above)
Comment (themuflon) [inline loom/graphics/gfxQuadRenderer.cpp]: Ask the compiler if it inlines it :) It's just nicer in any case

Comment (SmilyOrg) [inline loom/graphics/gfxQuadRenderer.cpp]: For reference `nanovg_lm_gl.h` uses `#define LGL GFX::Graphics::context()`

Comment (bengarney) [inline loom/graphics/gfxQuadRenderer.cpp]: OK works for me. Using the #define consistently might be smart but is another task for another day.

**Pull request merged**